b9/core.cpp

```diff
@@ -26,11 +26,25 @@ void ExecutionContext::functionCall(Parameter value) {
   push(result);
 }
 
+void ExecutionContext::functionReturn(StackElement returnVal) {
+  // TODO
+}
+
 void ExecutionContext::primitiveCall(Parameter value) {
   PrimitiveFunction *primitive = virtualMachine_->getPrimitive(value);
   (*primitive)(this);
 }
 
+void ExecutionContext::jmp(Parameter offset) {
```

Comment:
The body is going to be `return offset`. Which I guess is fine, and having this function makes the code more consistent.